### ISSUE TITLE
The `LagAssist` plugin is encountering a `NoSuchFieldException` during its initialization process.

### ISSUE THREAD

**Issue by OtakuSweett** [alvindimas05/LagAssist#4]
**Issue:**

The `LagAssist` plugin is encountering a `NoSuchFieldException` during its initialization process.

**Stack Trace:**

```
[10:20:34 WARN]: java.lang.NoSuchFieldException: receiving
[10:20:34 WARN]:    at java.base/java.lang.Class.getField(Class.java:2284)
[10:20:34 WARN]:    at io.papermc.reflectionrewriter.runtime.AbstractDefaultRulesReflectionProxy.getField(AbstractDefaultRulesReflectionProxy.java:85)
[10:20:34 WARN]:    at io.papermc.paper.pluginremap.reflect.PaperReflectionHolder.getField(Unknown Source)
[10:20:34 WARN]:    at lagassist-2.31-1.21.1 (1).jar//org.alvindimas05.lagassist.packets.PacketInjector.Enabler(PacketInjector.java:29)
[10:20:34 WARN]:    at lagassist-2.31-1.21.1 (1).jar//org.alvindimas05.lagassist.packets.PacketMain.Enabler(PacketMain.java:18)
[10:20:34 WARN]:    at lagassist-2.31-1.21.1 (1).jar//org.alvindimas05.lagassist.Main.EnableClasses(Main.java:102)
[10:20:34 WARN]:    at lagassist-2.31-1.21.1 (1).jar//org.alvindimas05.lagassist.Main.onEnable(Main.java:67)
[10:20:34 WARN]:    at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:288)
[10:20:34 WARN]:    at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:203)
[10:20:34 WARN]:    at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109)
[10:20:34 WARN]:    at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520)
[10:20:34 WARN]:    at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:643)
[10:20:34 WARN]:    at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:592)
[10:20:34 WARN]:    at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:760)
[10:20:34 WARN]:    at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:522)
[10:20:34 WARN]:    at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:351)
[10:20:34 WARN]:    at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1222)
[10:20:34 WARN]:    at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:335)
[10:20:34 WARN]:    at java.base/java.lang.Thread.run(Thread.java:1583)
```

**Environment Details:**

- **LagAssist Version:** 2.31
- **Minecraft Server:** UniverseSpigot 1.21.1-DEV-ver/1.21.1@9f5bdaf (Paper Fork)
- **Java Version:** OpenJDK 21 (21.0.3+9-LTS; Eclipse Adoptium Temurin-21.0.3+9)
- **Operating System:** Linux 5.15.0-118-generic (amd64)

Full ServerLog: https://mclo.gs/H2G6pTf

LagAssist Configuration: https://mclo.gs/62yRb71

**Comment by OtakuSweett** [comment]
```plaintext
[10:46:12 ERROR]: Could not pass event EntityDamageEvent to LagAssist v2.31
java.lang.IllegalArgumentException: Cannot spawn an entity from its CraftBukkit implementation class 'org.bukkit.craftbukkit.entity.CraftZombie' use the Bukkit class instead. You can get the Bukkit representation via Entity#getType()#getEntityClass()
        at org.bukkit.craftbukkit.CraftRegionAccessor.createEntity(CraftRegionAccessor.java:506) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:422) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:418) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:404) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at lagassist-2.31-1.21.1 (1).jar/org.alvindimas05.lagassist.stacker.StackManipulator.spawnDeadEntity(StackManipulator.java:94) ~[lagassist-2.31-1.21.1 (1).jar:?]
        at lagassist-2.31-1.21.1 (1).jar/org.alvindimas05.lagassist.stacker.StackManipulator.onDamage(StackManipulator.java:71) ~[lagassist-2.31-1.21.1 (1).jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor702.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:79) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:59) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:630) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEvent(CraftEventFactory.java:275) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:1207) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:1197) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:1187) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:1090) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleLivingEntityDamageEvent(CraftEventFactory.java:1245) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.handleEntityDamage(LivingEntity.java:2371) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.hurt(LivingEntity.java:1494) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.monster.Zombie.hurt(Zombie.java:324) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Entity.baseTick(Entity.java:943) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.baseTick(LivingEntity.java:444) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Mob.baseTick(Mob.java:373) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Entity.tick(Entity.java:895) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.tick(LivingEntity.java:3177) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Mob.tick(Mob.java:450) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.monster.Zombie.tick(Zombie.java:243) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.tickNonPassenger(ServerLevel.java:1277) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.lambda$tick$4(ServerLevel.java:750) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:39) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:727) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1820) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:499) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1612) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1312) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:335) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
[10:46:15 INFO]: Villager Villager['Villager'/521, uuid='bb52ac30-74ed-413f-94e1-2ddeac8823be', l='ServerLevel[world]', x=62.75, y=70.00, z=74.34, cpos=[3, 4], tl=545, v=true] died, message: 'Villager was slain by Zombie x4'
[10:46:16 ERROR]: Could not pass event EntityDamageEvent to LagAssist v2.31
java.lang.IllegalArgumentException: Cannot spawn an entity from its CraftBukkit implementation class 'org.bukkit.craftbukkit.entity.CraftZombie' use the Bukkit class instead. You can get the Bukkit representation via Entity#getType()#getEntityClass()
        at org.bukkit.craftbukkit.CraftRegionAccessor.createEntity(CraftRegionAccessor.java:506) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:422) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:418) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:404) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at lagassist-2.31-1.21.1 (1).jar/org.alvindimas05.lagassist.stacker.StackManipulator.spawnDeadEntity(StackManipulator.java:94) ~[lagassist-2.31-1.21.1 (1).jar:?]
        at lagassist-2.31-1.21.1 (1).jar/org.alvindimas05.lagassist.stacker.StackManipulator.onDamage(StackManipulator.java:71) ~[lagassist-2.31-1.21.1 (1).jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor702.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:79) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:59) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:630) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEvent(CraftEventFactory.java:275) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:1207) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:1197) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:1187) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:1090) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleLivingEntityDamageEvent(CraftEventFactory.java:1245) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.handleEntityDamage(LivingEntity.java:2371) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.hurt(LivingEntity.java:1508) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.monster.Zombie.hurt(Zombie.java:324) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Entity.baseTick(Entity.java:943) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.baseTick(LivingEntity.java:444) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Mob.baseTick(Mob.java:373) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Entity.tick(Entity.java:895) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.tick(LivingEntity.java:3177) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Mob.tick(Mob.java:450) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.monster.Zombie.tick(Zombie.java:243) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.tickNonPassenger(ServerLevel.java:1277) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.lambda$tick$4(ServerLevel.java:750) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:39) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:727) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1820) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:499) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1612) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1312) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:335) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
[10:46:19 ERROR]: Could not pass event EntityDamageEvent to LagAssist v2.31
java.lang.IllegalArgumentException: Cannot spawn an entity from its CraftBukkit implementation class 'org.bukkit.craftbukkit.entity.CraftZombie' use the Bukkit class instead. You can get the Bukkit representation via Entity#getType()#getEntityClass()
        at org.bukkit.craftbukkit.CraftRegionAccessor.createEntity(CraftRegionAccessor.java:506) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:422) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:418) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:404) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at lagassist-2.31-1.21.1 (1).jar/org.alvindimas05.lagassist.stacker.StackManipulator.spawnDeadEntity(StackManipulator.java:94) ~[lagassist-2.31-1.21.1 (1).jar:?]
        at lagassist-2.31-1.21.1 (1).jar/org.alvindimas05.lagassist.stacker.StackManipulator.onDamage(StackManipulator.java:71) ~[lagassist-2.31-1.21.1 (1).jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor702.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:79) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:59) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:630) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEvent(CraftEventFactory.java:275) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:1207) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:1197) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:1187) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:1090) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleLivingEntityDamageEvent(CraftEventFactory.java:1245) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.handleEntityDamage(LivingEntity.java:2371) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.hurt(LivingEntity.java:1508) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.monster.Zombie.hurt(Zombie.java:324) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Entity.baseTick(Entity.java:943) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.baseTick(LivingEntity.java:444) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Mob.baseTick(Mob.java:373) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Entity.tick(Entity.java:895) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.tick(LivingEntity.java:3177) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Mob.tick(Mob.java:450) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.monster.Zombie.tick(Zombie.java:243) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.tickNonPassenger(ServerLevel.java:1277) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.lambda$tick$4(ServerLevel.java:750) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:39) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:727) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1820) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:499) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1612) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1312) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:335) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
[10:46:24 ERROR]: Could not pass event EntityDamageEvent to LagAssist v2.31
java.lang.IllegalArgumentException: Cannot spawn an entity from its CraftBukkit implementation class 'org.bukkit.craftbukkit.entity.CraftZombie' use the Bukkit class instead. You can get the Bukkit representation via Entity#getType()#getEntityClass()
        at org.bukkit.craftbukkit.CraftRegionAccessor.createEntity(CraftRegionAccessor.java:506) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:422) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:418) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.CraftRegionAccessor.spawn(CraftRegionAccessor.java:404) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at lagassist-2.31-1.21.1 (1).jar/org.alvindimas05.lagassist.stacker.StackManipulator.spawnDeadEntity(StackManipulator.java:94) ~[lagassist-2.31-1.21.1 (1).jar:?]
        at lagassist-2.31-1.21.1 (1).jar/org.alvindimas05.lagassist.stacker.StackManipulator.onDamage(StackManipulator.java:71) ~[lagassist-2.31-1.21.1 (1).jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor702.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:79) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:59) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:630) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEvent(CraftEventFactory.java:275) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:1207) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:1197) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:1187) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:1090) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at org.bukkit.craftbukkit.event.CraftEventFactory.handleLivingEntityDamageEvent(CraftEventFactory.java:1245) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.handleEntityDamage(LivingEntity.java:2371) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.hurt(LivingEntity.java:1494) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.monster.Zombie.hurt(Zombie.java:324) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Entity.baseTick(Entity.java:943) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.baseTick(LivingEntity.java:444) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Mob.baseTick(Mob.java:373) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Entity.tick(Entity.java:895) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.LivingEntity.tick(LivingEntity.java:3177) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.Mob.tick(Mob.java:450) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.entity.monster.Zombie.tick(Zombie.java:243) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.tickNonPassenger(ServerLevel.java:1277) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.lambda$tick$4(ServerLevel.java:750) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:39) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:727) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1820) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:499) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1612) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1312) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:335) ~[universe-1.21.1.jar:1.21.1-DEV-9f5bdaf]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
[10:46:35 INFO]: Villager Villager['Villager'/522, uuid='5e5c5bcd-10bf-4
```

**Comment by OtakuSweett** [comment]
I also have this error when a zombie attacks a villager

**Comment by alvindimas05** [comment]
Can you grab the 1.21 version from the artifact-jar.zip and try that version instead? Might be caused by Spigot not releasing 1.21.1 and I can't tested it yet.

https://github.com/alvindimas05/LagAssist/actions/runs/10366847879

**Comment by OtakuSweett** [comment]
The version I'm using is a fork of Paper

**Comment by OtakuSweett** [comment]
https://mclo.gs/69ZmeIi

I tried that version and exactly the same thing happens, I am not using Bukkit or Spigot, it is a fork of purpur/paper

**Comment by alvindimas05** [comment]
Try this build.

https://github.com/alvindimas05/LagAssist/actions/runs/10373090855

**Comment by alvindimas05** [comment]
Fixed at a311fa5.
